canvas_account_admin_tools/static/canvas_account_admin_tools/js/services/ATRAPIService.js

```diff
@@ -0,0 +1,106 @@
+(function() {
+  var module = angular.module('ATRAPIModule', []);
```

Comment:
Yes indeed we can settle on some better naming. I think there are still some inconsistencies to the way various components are named as well -- e.g. services, factories, CamelCase, etc.